src/main/java/seedu/address/logic/commands/ViewCommand.java

```diff
@@ -0,0 +1,73 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.IsSameIdPredicate;
+import seedu.address.model.person.NameContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all persons in address book whose name contains any of the argument keywords.
+ * Keyword matching is case insensitive.
+ */
+public class ViewCommand extends Command {
+
+    public static final String COMMAND_WORD = "view";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Views the specific student using their ID or name\n"
+            + "Parameters: NAME or ID\n"
+            + "Example: " + COMMAND_WORD + " John OR " + COMMAND_WORD + " 12345";
+
+    private final NameContainsKeywordsPredicate namePredicate;
```

Comment:
I'm Currently using ViewCommand.java for viewing of summary stats, hmm maybe i should change it to StatsCommand ?

Reply:
I will try combining with view first! But view is getting really long so in the future we can abstract it out